docs/developer/process/integration-release.md

```diff
@@ -95,246 +74,7 @@ To see all checks that need to be released, run `ddev release show ready`.
-
-  To resolve this, you'll need to bootstrap metadata for every integration:
-
-  1. Checkout and pull the most recent version of the `master` branch.
-
-      ```
-      git checkout master
-      git pull
-      ```
-
-  1. Sign everything.
-
-      ```
-      ddev release make all --sign-only
-      ```
-
-      You may need to touch your Yubikey multiple times.
-
-  1. Push your branch to GitHub.
-  1. Manually trigger a build.
-
-      ```
-      git tag <USERNAME>bootstrap-1.0.0 -m <USERNAME>bootstrap-1.0.0
-      ```
-
-      The tag name is irrelevant, it just needs to look like an integration release. Gitlab doesn't sync
-      deleted tags, so any subsequent manual trigger tags will need to increment the version number.
-
-  1. Delete the branch and tag, locally and on GitHub.
-    
-- If a feature PR conflicting with the release PR is merged out of order.
-
-    The following is a possible sequence of events that can result in the build pipeline failing:
-    
-        1. A release PR is opened
-        2. A feature PR is opened and merged
-        3. The release PR is merged after the feature PR.
-        4. The release PR will not have updated and signed the feature PR's files, the released wheel will also not contain the changes from the feature PR.
-  
-    You may see an error like so:
-    
-    ```text
-    in_toto.exceptions.RuleVerificationError: 'DISALLOW *' matched the following artifacts: ['/shared/integrations-core/datadog_checks_dev/datadog_checks/dev/tooling/commands/ci/setup.py']
-    ```
-      
-    1. Verify whether the hash signed in `.in-toto/tag.<KEYID>.link`, [(see example)](https://github.com/DataDog/integrations-core/blob/9836c71f15a0cb93c63c1d2950dcdc28b49479a7/.in-toto/tag.57ce2495.link) matches what's on `master` for the artifact in question.
-        
-        To see the hash for the artifact, run the following `shasum` command (replace local file path):
-        
-        ```
-        shasum -a 256 datadog_checks_dev/datadog_checks/dev/tooling/commands/ci/setup.py
-        ```
-    
-    1. If any artifact mismatches, check out and pull the most recent version of the `master` branch.
-    
-        ```
-        git checkout master
-        git pull
-        ```
-            
-    1. Release the integration again with a new version, bump the version appropriately.
-    
-        ```
-        ddev release make <INTEGRATION> --version <VERSION>
-        ```
-    
-    1. Verify that the integration files are signed, and update the integration changelog to reflect the feature PR title in the following format.
-        
-        ```
-        * [<Changelog label>] <PR Title>. [See #<PR Number>](<Github PR link>).
-        ```
-       
-    1. After approval, merge PR to master for a new build to be triggered.
-  
-
-## Releasers
```

Comment:
Should we keep this section? I'm not sure how important it is to keep this for transparency of valid signers.

Reply:
It's not even up to date